fulp_modules/features/antagonists/monster_hunter/hunting_contract.dm

```diff
@@ -66,13 +68,17 @@
 		if("select")
 			if(isnull(params["item"]))
 				return
-			var/item = text2path(params["item"])
+			var/datum/hunter_weapons/item = text2path(params["item"])
+			if(!ispath(item))
+				return
 			selected_item = item
 			. = TRUE
 			purchase(selected_item, usr)
 		if("claim_reward")
+			if(!objectives_completed || !length(owner.rabbits) || used_up)
```

Comment:
```suggestion
			if(!objectives_completed || length(owner.rabbits) || used_up)
```

Reply:
This would mean having more than 0 rabbits would make it impossible to claim, that's intentional?

Reply:
yeah, we initially have a full list of rabbits. Whenever a rabbit is found, its deleted from that list. if the length is 0 that would imply there are no more rabbits to be found